tests/harvest/test_scenarios.py

```
@@ -120,10 +120,15 @@ def test_scenario_one(
-            config,
-            store,
-        )
+        if store == ():
+            travharv = Harvest(
+                str(config),
+            )
+        else:
+            travharv = Harvest(
+                str(config),
+                store,
+            )
```

Ruff (0.8.2)

Use ternary operator `travharv = Harvest(str(config)) if store == () else Harvest(str(config), store)` instead of `if`-`else`-block

Replace `if`-`else`-block with `travharv = Harvest(str(config)) if store == () else Harvest(str(config), store)`

(SIM108)